model-optimizer/mo/front/kaldi/extractors/copy_ext.py

```diff
@@ -41,8 +41,9 @@ def replace_op(self, graph: Graph, node: Node):
                       }
         indexes_node = Const(graph).create_node(attrs=const_attrs)
 
-        perm_in_1 = Const(graph, {'value': np.array([1, 0], dtype=np.int64), 'shape': [2], 'data_type': np.int64}).create_node()
-        axis_const = Const(graph, {'value': int64_array(0)}).create_node()
+        perm_in_1 = Const(graph, {'value': np.array([1, 0], dtype=np.int64), 'shape': [2], 'data_type': np.int64,
+                                  'name': 'input_order/{}'.format(node.id)}).create_node()
+        axis_const = Const(graph, {'value': int64_array(0), 'name': 'axis/{}'.format(node.id)}).create_node()
```

Comment:
Use `create_op_node_with_second_input` to create `gather_node` node. The `axis_const` is used in the Gather op only.

Reply:
Done.